common/api-review/auth-exp.api.md

```diff
@@ -20,18 +20,18 @@ export interface ActionCodeInfo {
         multiFactorInfo?: MultiFactorInfo | null;
         previousEmail?: string | null;
     };
-    operation: ActionCodeOperation;
+    operation: string;

```

Comment:
We can use string literal unions to make it more type safe, using something like 
```
typeof ActionCodeOperation[keyof typeof ActionCodeOperation]
```

We can apply the same to other places.

Reply:
Done (where appropriate)

`ProviderId` and `SignInMethod` should remain `string`. Updated the others.